.github/workflows/scripts/add-codeowners-to-pr.sh

```diff
@@ -40,16 +40,17 @@ main () {
     # is only available through the API. To cut down on API calls to GitHub,
     # we use the latest reviews to determine which users to filter out.
     JSON=$(gh pr view "${PR}" --json "files,author,latestReviews" | tr -dc '[:print:]' | sed -E 's/\\[a-z]//g')
-    AUTHOR=$(printf "${JSON}"| jq -r '.author.login')
-    FILES=$(printf "${JSON}"| jq -r '.files[].path')
-    REVIEW_LOGINS=$(printf "${JSON}"| jq -r '.latestReviews[].author.login')
+    AUTHOR=$(printf "%s" "$JSON"| jq -r '.author.login')
+    FILES=$(printf "%s" "$JSON"| jq -r '.files[].path')
+    REVIEW_LOGINS=$(printf "%s" "$JSON"| jq -r '.latestReviews[].author.login')
```

Comment:
Why not just use `echo` (or maybe ignore the warning)? See also line 71 below

Reply:
This notation was suggested by Shellcheck only,

Reply:
What I meant is that I believe `printf "%s" "$JSON"` should be equivalent to `echo "$JSON"` and I think the latter is simpler, so I think we should use `echo "$JSON"` instead. Does that make sense?

Shellcheck does not suggest this, but it's not wrong to use `echo` in general, and AFAICT it should be accepted by it.